examples/ui/text_wrap.rs

```diff
@@ -0,0 +1,107 @@
+//! This example demonstrates text wrapping and use of the `LineBreakOn` property.
+
+use bevy::prelude::*;
+use bevy::text::BreakLineOn;
+use bevy::winit::WinitSettings;
+
+fn main() {
+    App::new()
+        .add_plugins(DefaultPlugins)
+        .insert_resource(WinitSettings::desktop_app())
+        .add_startup_system(spawn)
+        .run();
+}
+
+fn spawn(mut commands: Commands, asset_server: Res<AssetServer>) {
+    commands.spawn(Camera2dBundle::default());
+
+    let text_style = TextStyle {
+        font: asset_server.load("fonts/FiraSans-Bold.ttf"),
+        font_size: 14.0,
+        color: Color::WHITE,
+        ..Default::default()
```

Comment:
Other minor typo, `..default()` isn't necessary here as you've filled all the fields.